packages/story-editor/src/components/floatingMenu/layer.js

```diff
@@ -50,11 +55,35 @@ function FloatingMenuLayer() {
       workspaceHeight,
     })
   );
+  // We need to know if anything (non-bg) is selected, if more than one element is selected,
+  // the list of different element types selected as well as a unique identifier
+  // for the combined selection (in order to undismiss in case of change of seletion)
+  const {
+    hasSelection,
+    hasMultiSelection,
+    selectionTypes,
+    selectionIdentifier,
+  } = useStory(({ state: { selectedElements } }) => ({
+    hasSelection:
+      selectedElements.filter(({ isBackground }) => !isBackground).length > 0,
+    hasMultiSelection: selectedElements.length > 1,
+    selectionTypes: [
```

Comment:
Maybe `selectedTypes`? In case this is for storing the types of elements that are selected. Otherwise it sounds like "types of selection" which isn't clear to understand. Maybe even `selectedElementTypes` for making it easier to understand for people in the future 😛

Reply:
+1
`selectedElements.map(select type) = selectedElementsTypes` makes sense.

Reply:
I'll changed it to `selectedElementTypes`.